db/migrate/20171212214057_create_reports.rb

```diff
@@ -0,0 +1,26 @@
+class CreateReports < ActiveRecord::Migration[5.1]
+  def change
+    create_table :reports do |t|
+      t.string :ref_no, limit: 50, null: false
```

Comment:
It should be nullable as initially submitted records won't have ref_no. They are later updated from `@vilys`.